### PR TITLE
Removed a check that was discarding valid matrices

### PR DESCRIPTION
**Pull request by chevtche**

### CODE REVIEW

**Comment by chevtche** [comment]
If the numbers in the matrix are small, the determinant is small and the matrix is discarded, being valid...

**Comment by rdumusc** [comment]
bum patch version, update changelog, add a unit test for a previously non-invertible matrix?

File: doc/Changelog.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 # git master
 
+* [62](https://github.com/Eyescale/vmmlib/pull/62)"
```
Comment (rdumusc) [inline doc/Changelog.md]: rm "

File: doc/Changelog.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
...
+  Removed a check that was discarding valid matrices
```
Comment (rdumusc) [inline doc/Changelog.md]: ... a check in Matrix::computeInverse ...

**Comment by rdumusc** [comment]
+1

**Pull request merged**